### PR TITLE
[UTOPIA-926] move mannual change status to subheader and update intake view page 

### PR DESCRIPTION
**Pull request by binggaofw**
# Description

This PR includes the following proposed change(s):

- move manual status change to subheader component
- update intake view page follow figma

Screen Shot 2023-03-15 at 12 22 57 PM
Screen Shot 2023-03-15 at 12 23 03 PM
Screen Shot 2023-03-15 at 12 23 08 PM

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

**Review by antsand — APPROVED** [review]

File: src/frontend/src/components/public/PIAFormTabs/intake/index.tsx [hunk 1]
```diff
@@ -197,111 +197,178 @@ export const PIAFormIntake = () => {
...
-            <MDEditor
-              preview={isReadOnly ? 'preview' : 'edit'}
-              value={intakeForm?.initiativeDescription}
-              onChange={(value) =>
-                stateChangeHandler(value, 'initiativeDescription')
-              }
-            />
+            {(isReadOnly && !intakeForm.initiativeDescription) ||
+            (isReadOnly && intakeForm.initiativeDescription === '') ? (
+              <p>Not answered</p>
```
Comment (Bottle7) [inline src/frontend/src/components/public/PIAFormTabs/intake/index.tsx:219]:
```suggestion
              <p><i>Not answered</i></p>
```

File: src/frontend/src/components/public/PIAFormTabs/intake/index.tsx [hunk 2]
```diff
@@ -197,111 +197,178 @@ export const PIAFormIntake = () => {
...
         <div className="richText" id="initiativeScope">
-          <MDEditor
-            preview={isReadOnly ? 'preview' : 'edit'}
-            value={intakeForm?.initiativeScope}
-            defaultTabEnable={true}
-            onChange={(value) => stateChangeHandler(value, 'initiativeScope')}
-          />
+          {(isReadOnly && !intakeForm.initiativeScope) ||
+          (isReadOnly && intakeForm.initiativeScope === '') ? (
+            <p>Not answered</p>
```
Comment (Bottle7) [inline src/frontend/src/components/public/PIAFormTabs/intake/index.tsx:251]:
```suggestion
            <p><i>Not answered</i></p>
```

File: src/frontend/src/components/public/PIAFormTabs/intake/index.tsx [hunk 3]
```diff
@@ -197,111 +197,178 @@ export const PIAFormIntake = () => {
...
-            preview={isReadOnly ? 'preview' : 'edit'}
-            value={intakeForm?.dataElementsInvolved}
-            defaultTabEnable={true}
-            onChange={(value) =>
-              stateChangeHandler(value, 'dataElementsInvolved')
-            }
-          />
+          {(isReadOnly && !intakeForm.dataElementsInvolved) ||
+          (isReadOnly && intakeForm.dataElementsInvolved === '') ? (
+            <p>Not answered</p>
```
Comment (Bottle7) [inline src/frontend/src/components/public/PIAFormTabs/intake/index.tsx:280]:
```suggestion
            <p><i>Not answered</i></p>
```

File: src/frontend/src/components/public/PIAFormTabs/intake/index.tsx [hunk 4]
```diff
@@ -197,111 +197,178 @@ export const PIAFormIntake = () => {
...
-                  preview={isReadOnly ? 'preview' : 'edit'}
-                  value={intakeForm?.riskMitigation}
-                  defaultTabEnable={true}
-                  onChange={(value) =>
-                    stateChangeHandler(value, 'riskMitigation')
-                  }
-                />
+                {(isReadOnly && !intakeForm.riskMitigation) ||
+                (isReadOnly && intakeForm.riskMitigation === '') ? (
+                  <p>Not answered</p>
```
Comment (Bottle7) [inline src/frontend/src/components/public/PIAFormTabs/intake/index.tsx:359]:
```suggestion
                  <p><i>Not answered</i></p>
```

**Comment by Bottle7** [comment]
Other than the suggestions I left, this looks good to me 😄 🚀

**Review by Bottle7 — APPROVED** [review]
🚀

**Review by Bottle7 — APPROVED** [review]

**Review by kushal-arora-fw — APPROVED** [review]
Ok for now. UTOPIA-934 should fix something.

**Review by antsand — APPROVED** [review]

**Pull request merged**